src/main/java/io/xpring/ClassicAddress.java

```diff
@@ -0,0 +1,45 @@
+package io.xpring;
+
+import io.xpring.javascript.JavaScriptWallet;
+import io.xpring.javascript.JavaScriptWalletFactory;
+import io.xpring.javascript.JavaScriptWalletGenerationResult;
+
+/**
+ * Represents classic address components on the XRP Ledger.
+ */
+public class ClassicAddress {
```

Comment:
Any thoughts on adding equals/hashcode implementations to this class? I don't know if you've used it a lot or have any desire to, but the Java library Immutables makes this a bit easier and works well for simple immutable types like this.

Reply:
@keefertaylor if you want to hop on a bluejeans I can show you how immutables works - it's pretty easy to grok, and will save you a lot of pain here.

Reply:
Done.